dotcom-rendering/src/components/BylineLink.tsx

```diff
@@ -104,28 +109,28 @@ const ContributorLink = ({
 	</a>
 );
 
+const Source = ({ source }: { source: string }) => (
```

Comment:
Changes here come from this frontend code https://github.com/guardian/frontend/blob/main/common/app/model/content.scala#L761